packages/twenty-front/src/modules/object-record/record-field/utils/isFieldReadonlyFromObjectMetadataName.ts

```diff
@@ -0,0 +1,17 @@
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { isWorkflowSubObject } from '@/object-metadata/utils/isWorkflowSubObject';
+
+export const isFieldReadonlyFromObjectMetadataName = (
```

Comment:
I would put these hooks into object-metadata?

Reply:
checked with you. Leaving them here